src/containers/Login.jsx

```diff
@@ -1,43 +1,58 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { translate } from 'react-i18next';
+
 import Center from '../components/layouts/Center';
 
-class Login extends Component {
+export class Login extends Component {
   render() {
-    const { loginGoogle } = this.props;
+    const { loginGoogle, t, i18n } = this.props;
+
+    const changeLanguage = lng => {
+      i18n.changeLanguage(lng);
+    };
+
     return (
       <Center>
         <div className="loginscreen">
+          {/* This is just for demo purposes */}
+          <button type="button" onClick={() => changeLanguage('en')}>
+            es
```

Comment:
Should it be 'en' for english?

Reply:
actually yes, I was not thinking to have those buttons as the final version, so /* This is just for demo purposes */, we could have a dropdown in a corner, or other thing, but if you think those buttons are ok, I'm going to update them